yb-voyager/src/srcdb/data/gather-assessment-metadata/oracle/yb-voyager-oracle-gather-assessment-metadata.sh

```diff
@@ -140,6 +140,13 @@ parse_oracle_dsn() {
     echo "$ORACLE_DSN"
 }
 
+# Function to remove password from oracle connection string
+remove_password() {
+    local connection_string="$1"
+    local cleaned_string=$(echo "$connection_string" | sed -r 's|/[^@]*@|/*****@|')
```

Comment:
Used chatgpt to understand this :) Looks good. IIUC, everything between / and @ is replaced with *****, correct?

Reply:
yes right. I also took GPTs help for the solution :)